Camunda.Api.Client/Camunda.Api.Client.csproj

```diff
@@ -26,7 +26,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Refit" Version="5.0.23" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="Refit" Version="6.3.2" />
+    <PackageReference Include="Refit.Newtonsoft.Json" Version="6.3.2" />
+    <PackageReference Include="Refit.Xml" Version="6.3.2" />
```

Comment:
Refit 6 splits some of its functionality into separate packages, but we need all of these.